Dockerfile

```diff
@@ -73,5 +73,8 @@ RUN apk update \
 
 WORKDIR /repo
 
+RUN addgroup -S voxbox && adduser -S voxbox -G voxbox
```

Comment:
what do those parameters do? Is there maybe a long version that we could use? I think that makes the whole Dockerfile easier to read.

Reply:
nein! there are no long parameters in alpine. it may cost to much space! -S is for system

```
/ # addgroup --help
BusyBox v1.36.1 (2024-06-10 07:11:47 UTC) multi-call binary.

Usage: addgroup [-g GID] [-S] [USER] GROUP

Add a group or add a user to a group

	-g GID	Group id
	-S	Create a system group
/ # adduser --help
BusyBox v1.36.1 (2024-06-10 07:11:47 UTC) multi-call binary.

Usage: adduser [OPTIONS] USER [GROUP]

Create new user, or add USER to GROUP

	-h DIR		Home directory
	-g GECOS	GECOS field
	-s SHELL	Login shell
	-G GRP		Group
	-S		Create a system user
	-D		Don't assign a password
	-H		Don't create home directory
	-u UID		User id
	-k SKEL		Skeleton directory (/etc/skel)
/ #
```